incubator/ado-agent/templates/secret.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: vsts
+type: Opaque
+data:
+  account: {{ .Values.vsts.account }}
+  token: {{ .Values.vsts.token }}
+  pool: {{ .Values.vsts.pool }}
```

Comment:
The secrets need to be base64 encoded. You can see examples of doing this in charts at https://github.com/helm/charts/search?q=b64enc&unscoped_q=b64enc